persepolis.json

```diff
@@ -0,0 +1,50 @@
+{
+    "homepage": "https://persepolisdm.github.io/",
+    "description": "A Free as libre download manager & a GUI for aria2, written in Python",
+    "version": "3.1.0",
+    "license": "GPL-3.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/persepolisdm/persepolis/releases/download/3.1.0/persepolis_3_1_0_windows_64bit.exe",
+            "hash": "fe1fc842b49daa555af3d1ffce4f8f5a2b664620d5004b96cbb6f177ed7be29f"
+        },
+        "32bit": {
+            "url": "https://github.com/persepolisdm/persepolis/releases/download/3.1.0/persepolis_3_1_0_windows_32bit.exe",
+            "hash": "dde3b8b72dc903b369b95af641454f6fe74e8961cd2008fdc5a1427e88b39809"
+        }
+    },
+    "innosetup": true,
+    "bin": [
+        [
+            "Persepolis Download Manager.exe",
```

Comment:
Is it advisable to have spaces in `bin` executables?

Reply:
I already set non-space alias to it, for convenience to type. But it should not have any issue. If it does, IMHO the OS should be blamed for it, though Windows seems has done better than Unix-like here.